app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,76 @@
+class CommentsController < ApplicationController
+  before_action :set_micropost, only: [:create, :new]
```

Comment:
có load and authorize resouce rồi thì a nghĩ ko cần `set_micropost` + `set_comment`

Reply:
e sử dụng 2 dòng này ạ
load_and_authorize_resource :micropost
load_and_authorize_resource :comment, through: :micropost